code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -91,11 +91,21 @@
 	RegisterSignal(alert_control.listener, COMSIG_ALARM_LISTENER_TRIGGERED, PROC_REF(alarm_triggered))
 	RegisterSignal(alert_control.listener, COMSIG_ALARM_LISTENER_CLEARED, PROC_REF(alarm_cleared))
 
+	//Heads up to other binary chat listeners that a new AI is online and listening to Binary.
+	if(announce_init_to_others && !(z == 1)) //Skip new syndicate AIs and also new AIs on centcom Z
```

Comment:
```suggestion
	if(announce_init_to_others && z != 1) //Skip new syndicate AIs and also new AIs on centcom Z
```

You should probably check for centcom z level instead of a direct number comparasion